<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="username_label">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="username">
       <property name="text">
        <string>feng</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password">
       <property name="text">
        <string>anderson</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dmz_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>dmz_password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>601</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="HC_tab">
     <attribute name="title">
      <string>Health Check</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>591</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="update_all_button">
       <property name="text">
        <string>更新全部</string>
       </property>
      </widget>
      <widget class="QPushButton" name="update_selected_button">
       <property name="text">
        <string>更新选中行</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Log_tab">
     <attribute name="title">
      <string>Get Log</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Log</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="log_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="start_date_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Start date</string>
      </property>
     </widget>
     <widget class="QLabel" name="end_date_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>End date</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="start_date_Edit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1753</year>
        <month>9</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyyMMdd</string>
      </property>
      <property name="date">
       <date>
        <year>2022</year>
        <month>8</month>
        <day>10</day>
       </date>
      </property>
     </widget>
     <widget class="QDateEdit" name="end_date_Edit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyyMMdd</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Run Command</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Command</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>run</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
